tt

```diff
@@ -57,6 +55,10 @@ main() {
 		esac
 	fi
 
+	TT_PROJECTS=${TT_PROJECTS:-~/tmp}
+	TT_GIT_BASE=${TT_GIT_BASE:-git@github.com:meet-tarantino}
+	apply_workspace_vars # this may override the above defaults
```

Comment:
Now that I think of it, are you sure we need to do this explicitly? Part of our vision included allowing workspaces to define their own bash scripts to be automatically sourced so that additional commands could be added. (Think `tt logs` being added by a workspace where Kibana is included.)